packages/odyssey-react/src/components/Button/Button.theme.ts

```diff
@@ -13,109 +13,99 @@
+  DangerDisabledBorderColor: theme.ColorBorderDangerLight,
+  DangerFocusBackgroundColor: theme.ColorPaletteRed900,
+  DangerFocusBoxShadowColor: theme.ColorFocusDanger,
+  DangerHoverBackgroundColor: theme.ColorPaletteRed900,
+  DangerHoverBorderColor: theme.ColorBorderDangerDark,
 
   // Dismiss Variant
-  ColorBackgroundDismiss: "transparent",
-  ColorBorderDismissHover: "transparent",
-  ColorTextDismiss: "inherit",
-  ColorTextDismissDisabled: theme.ColorNeutralBase,
-  FontLineHeightDismiss: 1,
-  SpacePaddingBlockDismiss: theme.SpaceEmXs,
-  SpacePaddingInlineDismiss: theme.SpaceEmXs,
-  ColorBackgroundDismissHover: "rgba(255, 255, 255, 0.6)",
-  ColorBackgroundDismissFocus: "rgba(255, 255, 255, 0.6)",
-  ColorBackgroundDismissDisabled: "rgba(235, 235, 237, 0.6)",
+  DismissBackgroundColor: "transparent",
+  DismissHoverBorderColor: "transparent",
+  DismissTextColor: "inherit",
+  DismissDisabledTextColor: theme.ColorNeutralBase,
+  DismissLineHeight: 1,
+  DismissPaddingBlock: theme.SpaceEmXs,
+  DismissPaddingInline: theme.SpaceEmXs,
+  DismissHoverBackgroundColor: "rgba(255, 255, 255, 0.6)",
+  DismissFocusBackgroundColor: "rgba(255, 255, 255, 0.6)",
+  DismissDisabledBackgroundColor: "rgba(235, 235, 237, 0.6)",
 
   // Dismiss Inverted Variant
-  ColorShadowDismissInverted: theme.ColorPaletteNeutralWhite,
+  DismissInvertedBoxShadowColor: theme.ColorPaletteNeutralWhite,
 
   // Clear Variant
-  ColorBackgroundClear: "transparent",
-  ColorBackgroundClearDisabled: "transparent",
-  ColorBackgroundClearFocus: theme.ColorBackgroundBase,
-  ColorBackgroundClearHover: theme.ColorBackgroundPrimaryLight,
-  ColorBorderClearHover: "transparent",
-  ColorTextClear: theme.ColorPrimaryBase,
-  ColorTextClearDisabled: theme.ColorPrimaryLight,
-  ColorTextClearFocus: theme.ColorTextPrimary,
-  ColorTextClearHover: theme.ColorPaletteBlue900,
+  ClearBackgroundColor: "transparent",
+  ClearDisabledBackgroundColor: "transparent",
+  ClearDisabledTextColor: theme.ColorPrimaryLight,
+  ClearFocusBackgroundColor: theme.ColorBackgroundBase,
+  ClearFocusTextColor: theme.ColorTextPrimary,
+  ClearHoverBackgroundColor: theme.ColorBackgroundPrimaryLight,
+  ClearHoverBorderColor: "transparent",
+  ClearHoverTextColor: theme.ColorPaletteBlue900,
+  ClearTextColor: theme.ColorPrimaryBase,
 
   // Wide Layout
-  SpaceMarginBlockEndWideLayout: theme.SpaceRemS,
-  SpaceMarginBlockWideLayout: 0,
-  SpaceMarginInlineWideLayout: 0,
+  WideLayoutMarginBlock: 0,
+  WideLayoutMarginBlockEnd: theme.SpaceRemS,
+  WideLayoutMarginInline: 0,
 
   // Small Size
-  FontLineHeightSmallSize: theme.FontLineHeightHeading,
-  FontSizeSmallSize: theme.FontSizeCaption,
+  SmallSizeFontSize: theme.FontSizeCaption,
+  SmallSizeLineHeight: theme.FontLineHeightHeading,
 
   // Large Size
-  SpacePaddingBlockLargeSize: theme.SpaceEmS,
-  SpacePaddingInlineLargeSize: theme.SpaceEmM,
-});
-
-const root: ThemeReducer = (theme) => ({
```

Comment:
+1 I was experimenting a bit with how to organize tokens but no reason for Button to do things differently than all the others yet